lib/storage/s3_storage.ex

```diff
@@ -23,10 +23,13 @@ defmodule Uploadex.S3Storage do
   alias ExAws.S3
 
   @impl true
-  def store(%{filename: filename, path: path}, opts) do
+  def store(%{filename: filename, path: path, content_type: content_type}, opts) do
     bucket = Keyword.fetch!(opts, :bucket)
     directory = Keyword.fetch!(opts, :directory)
-    upload_opts = Keyword.get(opts, :upload_opts, [])
+    upload_opts =
+      opts
+      |> Keyword.get(:upload_opts, [])
+      |> Keyword.put_new(:content_type, content_type)
```

Comment:
Please document that we can pass `content_type` as upload_opts for S3 storage, and that the default is the upload's content type